<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>201</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>节点数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>接受数据</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_state">
    <property name="geometry">
     <rect>
      <x>327</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>未连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>570</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>提供服务</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_app">
    <property name="geometry">
     <rect>
      <x>327</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>未连接</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>路由器连接状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>app连接状态</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_table1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>送一号桌</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_table2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>送二号桌</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_table3">
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>350</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>送三号桌</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_table4">
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>350</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>送四号桌</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
